feature(strict_color_names): Added a rule for color validation. (#60)
lyft/xiblint: README.md

- `named_colors`

>   Ensures all colors are using named colors from an asset catalog. Configure `allowed_colors` (allowed set of color names), `allow_system_colors` (default is `false`) and `ignore_alpha` (default is `false`) in a
>   custom rule configuration using `rules_config` (see below) if you’d like to ignore colors with alpha.

Reviewer: ```suggestion
  Ensures all colors are using named colors from an asset catalog. Configure `allowed_colors` to limit the colors to a subset (default is all named colors are allowed), `allow_system_colors` (default is `false`) and `ignore_alpha` (default is `false`) in a custom rule configuration using `rules_config` (see below).
```
